jvm-runtime/ftl-runtime/pom.xml

```diff
@@ -293,5 +368,122 @@
+             and sets up the distribution management for both snapshot and release repositories.
+             Activate this profile when deploying open-source projects to Maven Central. -->
+        <profile>
+            <!-- https://central.sonatype.org/publish/publish-maven/#deployment -->
+            <id>ossrh</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.13</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <serverId>ossrh-releases</serverId>
+                            <nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
+                            <autoReleaseAfterClose>true</autoReleaseAfterClose>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+            <!-- specifies where the project artifacts should be distributed or deployed. -->
+            <!-- tells Maven where to deploy your project's artifacts when you run mvn deploy -->
+            <distributionManagement>
+                <snapshotRepository>
+                    <id>ossrh-snapshots</id>
+                    <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+                </snapshotRepository>
+                <!-- Sonatype's OSSRH - replicates to Maven Central within 30min of publish -->
+                <repository>
+                    <id>ossrh-releases</id>
+                    <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+                </repository>
+            </distributionManagement>
+        </profile>
     </profiles>
+
+    <!-- specifies where the project artifacts should be distributed or deployed.
+         tells Maven where to deploy your project's artifacts when you run mvn deploy -->
+    <distributionManagement>
+        <!-- primary release repository. not being used yet -->
+        <repository>
+            <uniqueVersion>false</uniqueVersion>
+            <id>tbd-oss-releases</id>
+            <name>TBD OSS Releases Repository</name>
+            <url>https://blockxyz.jfrog.io/artifactory/tbd-oss-releases-maven2</url>
+            <layout>default</layout>
+        </repository>
+        <!-- primary snapshot repository for all "*-SNAPSHOT" artifacts. not yet being used -->
+        <snapshotRepository>
+            <uniqueVersion>false</uniqueVersion>
+            <id>tbd-oss-snapshots</id>
+            <name>TBD OSS Snapshots Repository</name>
+            <url>https://blockxyz.jfrog.io/artifactory/tbd-oss-snapshots-maven2</url>
+            <layout>default</layout>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <!-- additional repositories that we can pull dependencies from -->
+    <repositories>
```

Comment:
In general this is a bad idea to put TBH repos into a publicly published pom. This will mean that projects consuming this pom will end up with these repositories in their repo list. If we really need them they should be in a 
`settings.xml` file.

Reply:
@stuartwdouglas it may be best to connect with @ALRubinger as he paved the way here for other jvm artifacts at tbd. Take a look at web5-kt or tbdex-kt for reference. 

No real opinions from my pov especially with limited jvm experience. Happy to do whatever you think makes sense.

Reply:
I have moved it to a settings.xml profile for now.